plenty_api/api.py

```diff
@@ -614,12 +613,14 @@ def plenty_api_get_bi_raw_files(self, refine: dict, download=False, download_dir
             logging.error("GET BI-Rawfile list failed with:\n"
                           f"{bi_files}")
             return None
-            
-        if download:
-            for bi_file in bi_files:
-                file_destination = os.path.join(download_directory, os.path.basename(bi_file['path']))
-                self.__get_request_dump_file(domain='bi_raw', path='/file', query={'path':bi_file['path']}, download_filepath=file_destination)
 
+        if download:
+            if len(bi_files) > 0:
+                for bi_file in bi_files:
+                    file_destination = os.path.join(download_directory, os.path.basename(bi_file['path']))
+                    self.__get_request_dump_file(domain='bi_raw', path='/file', query={'path':bi_file['path']}, download_filepath=file_destination)
+            else:
+                logging.warn('Nothing to download - got empty result! Ether check filter params or BI config in PlentyMarkets!')
```

Comment:
s/Ether/Either/

And you can lose a level of indentation with:
```python
            if not len(bi_files):
                logging.warn('Nothing to download - got empty result! Ether check filter params or BI config in PlentyMarkets!')
            for bi_file in bi_files:
                ....
```

And finally please make sure that the logging line doesn't exceed 100 characters.